module-bluetooth/Bluetooth/BluetoothStateMachine.hpp

```diff
@@ -355,6 +355,7 @@ namespace bluetooth
                         state<Idle> + sml::event<bluetooth::event::VisibilityOn> / HandleSetVisibility = state<Idle>,
                         state<Idle> + sml::event<bluetooth::event::VisibilityOff> / HandleUnsetVisibility = state<Idle>,
                         state<Idle> + sml::event<bluetooth::event::ConnectAudio> / EstablishAudioConnection = state<Idle>,
+                        state<Idle> + sml::event<bluetooth::event::DisconnectAudio> / HandleDisconnect = state<Idle>,
```

Comment:
UT or didn't happen

Reply:
+1

Reply:
Added UT and renamed the connect/disconnect events